content/en/platform/corda/5.0/developing-applications/basic-cordapp/_index.md

```diff
@@ -1,13 +1,13 @@
 ---
 date: '2023-02-23'
-title: "Building Your First Basic CorDapp"
+title: "Building a Basic CorDapp"
 project: corda
 version: 'Corda 5.0'
 menu:
   corda5:
     identifier: corda5-develop-first-cordapp
     parent: corda5-develop
-    weight: 5000
+    weight: 6000
 section_menu: corda5
 ---
 _direct lift of existing section?_:
```

Comment:
```suggestion
_direct lift of existing section?_:
```

Just an image and this comment in the file.

Reply:
Yes, this is just a placeholder, for now.